src/main/java/Dice.java

```diff
@@ -0,0 +1,20 @@
+import java.util.*;
+
+public class Dice {
+
+    private Random random;
+
+    public Dice(Random random) {
+        this.random = random;
+    }
+
+    public int roll_dice() {
+        return random.nextInt(6);
```

Comment:
Can we add one to the result of `rand.nextInt()`? 6 is not included in the possible values that can be returned because the `nextInt` input is exclusive.

Reply:
Good catch! I'll add the +1.